main.go

```diff
@@ -0,0 +1,18 @@
+// Package main is the entry point for the application.
+package main
+
+import (
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/ubuntu/ubuntu-proxy-manager/internal/app"
+)
+
+func main() {
+	app, err := app.New()
```

Comment:
I agree that parsing argument with cobra will be way overkill for this. However, I strongly think that no program should run with `--help` and should print an usage if badly used (like passing an argument when expecting none). Can you handle that manually, or using pflags for posix-like flags?